app/controllers/action_officers_controller.rb

```diff
@@ -2,11 +2,19 @@ class ActionOfficersController < ApplicationController
   before_action :authenticate_user!, PQUserFilter
 
   def index
-    @action_officers = ActionOfficer.active_list
-                                    .joins(deputy_director: :division)
-                                    .order(deleted: :asc)
-                                    .order(Arel.sql('lower(divisions.name)'))
-                                    .order(Arel.sql('lower(action_officers.name)'))
+    @show_inactive = (params[:show_inactive] == 'true')
```

Comment:
I don't think you need parentheses here. They are useful when there are multiple conditions, but as this is a simple equality check I don't think they add anything here.

Reply:
I have left the parentheses as it helps me to understand the @show_inactive value